components/typography/typography.js

```diff
@@ -141,6 +141,28 @@ if (!document.head.querySelector('#d2l-typography-font-face')) {
 			}
 
 		}
+
+		.d2l-typography .d2l-blockquote {
+			background-image: url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTEiIGhlaWdodD0iMTEiIHZpZXdCb3g9IjAgMCAyMiAyMiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+PGRlZnM+PHBhdGggaWQ9ImEiIGQ9Ik0wIDBoMjR2MjRIMHoiLz48L2RlZnM+PGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoLTEgLTEpIiBmaWxsPSJub25lIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiPjxtYXNrIGlkPSJiIiBmaWxsPSIjZmZmIj48dXNlIHhsaW5rOmhyZWY9IiNhIi8+PC9tYXNrPjxwYXRoIGQ9Ik02IDIyLjY2N0E0LjY2NyA0LjY2NyAwIDAgMCAxMC42NjcgMThjMC0xLjIyNy0uNTU5LTIuNS0xLjMzNC0zLjMzM0M4LjQ4MSAxMy43NSA3LjM1IDEzLjMzMyA2IDEzLjMzM2MtLjQxMSAwIDEuMzMzLTYuNjY2IDMtOSAxLjY2Ny0yLjMzMyAxLjMzMy0zIC4zMzMtM0M4IDEuMzMzIDUuMjUzIDQuNTg2IDQgNy4yNTUgMS43NzMgMTIgMS4zMzMgMTUuMzkyIDEuMzMzIDE4QTQuNjY3IDQuNjY3IDAgMCAwIDYgMjIuNjY3em0xMiAwQTQuNjY3IDQuNjY3IDAgMCAwIDIyLjY2NyAxOGMwLTEuMjI3LS41NTktMi41LTEuMzM0LTMuMzMzLS44NTItLjkxNy0xLjk4My0xLjMzNC0zLjMzMy0xLjMzNC0uNDExIDAgMS4zMzMtNi42NjYgMy05IDEuNjY3LTIuMzMzIDEuMzMzLTMgLjMzMy0zLTEuMzMzIDAtNC4wOCAzLjI1My01LjMzMyA1LjkyMkMxMy43NzMgMTIgMTMuMzMzIDE1LjM5MiAxMy4zMzMgMThBNC42NjcgNC42NjcgMCAwIDAgMTggMjIuNjY3eiIgZmlsbD0iI0QzRDlFMyIgbWFzaz0idXJsKCNiKSIvPjwvZz48L3N2Zz4=");
+			background-position: left top;
+			background-repeat: no-repeat;
+			color: var(--d2l-color-tungsten);
+			font-size: 0.8rem;
+			font-weight: 400;
+			line-height: 1.2rem;
+			margin-left: 0;
+			margin-right: 1.2rem;
+			padding-left: 1.2rem;
+			padding-top: 0.2rem;
+		}
+
+		.d2l-typography .d2l-blockquote[dir="rtl"] {
```

Comment:
These styles will be in the global scope so the `blockquote` element wouldn't have a `dir="rtl"` attribute on it. So I think this needs to be `[dir="rtl"] .d2l-typography .d2l-blockquote`, given that we render `dir` on the `html` element.


Aside, in the web component case we sprout `dir="rtl"` on the host using the `RtlMixin`, and we do that because selectors inside of shadow roots can't be configured to select based on attributes in the parent DOM scope(s).

Reply:
The reason why the current selector works in the demo-snippet is because the demo-snippet has some code to set `dir="rtl"` directly on the element to simulate what `RtlMixin` does. While `[dir="rtl"] .d2l-typography .d2l-blockquote` is what we need for our app, it won't work for the demo-snippet RTL switch. To make it work in both scenarios we'd need:

``` css
[dir="rtl"] .d2l-typography .d2l-blockquote,
.d2l-typography .d2l-blockquote[dir="rtl"] {
    ...
}
```

Reply:
Ok will adjust